exercises/scale-generator/description.md

```diff
@@ -47,3 +58,12 @@ Non-diatonic scales can contain other intervals.  An "augmented second"
 interval, written "A", has two interceding notes (e.g., from A to C or Db to E)
 or a "whole step" plus a "half step". There are also smaller and larger
 intervals, but they will not figure into this exercise.
+
+### Task
+
+Given a tonic and a set of intervals, generate the musical scale starting with
+the tonic and following the specified interval pattern.
+
+This is similar to generating chromatic scales except that instead of returning
+12 notes you need to return one note for each interval in the given pattern by
+skipping the number of notes indicated by each interval in the pattern.
```

Comment:
Here is where I would move the sentence from above. But probably modified to something like this:

```suggestion
skipping the number of notes indicated by each interval in the pattern.

As a reminder, notes are still only ever referred to with uppercase letters.
Even though minor scales are sometimes named using lowercase letters (such as
"f# minor"), the first note of f# minor is still F# (written in uppercase), and 
all other notes in the scale are also written in uppercase.
```

Reply:
as I said above IMO this second task should only introduce the complexity of the pattern input. I think if you introduce this casing thing here students might be misled to try and deal with casing differently for the two tasks.